src/core/Akka.Remote.Tests/Transport/DotNettyTransportDnsResolutionSpec.cs

```diff
@@ -299,6 +299,20 @@ public Property HeliosTransport_should_map_valid_IPEndpoints_to_Address(IPEndPoi
             return endpoint.Equals(parsedEp).Label("Should be able to parse endpoint to address and back");
         }
 
+        /// <summary>
+        /// Testing public port
+        /// </summary>
+        /// <param name="endpoint"></param>
+        /// <returns></returns>
+        [Property]
+        public Property HeliosTransport_should_map_valid_IPEndpoints_to_Address_when_using_publicport(IPEndPoint endpoint)
```

Comment:
Doesn't really go far enough with the testing. Should be able to test all the way down to what gets extracted via configuration.